frontend/src/_styles/theme.scss

```diff
@@ -10956,7 +10956,7 @@ tbody {
     background: var(--base) !important;
     border: 1px solid var(--slate7) !important;
     border-radius: 6px !important;
-    margin-bottom: 0px !important;
+    margin-bottom: 4px !important;
```

Comment:
Is this change required? Could you please check if it will affect the design of other components?
> //TJ APP INPUT
> .tj-app-input,
> .edit-row-container {